sessionctx/variable/varsutil.go

```diff
@@ -635,6 +635,11 @@ func ValidateSetSystemVar(vars *SessionVars, name string, value string) (string,
 			return "", nil
 		}
 		return checkUInt64SystemVar(name, value, 1, math.MaxUint32, vars)
+	case TiDBStmtSummaryHistorySize:
+		if value == "" {
+			return "", nil
+		}
+		return checkUInt64SystemVar(name, value, 0, math.MaxInt8, vars)
```

Comment:
max value is `math.MaxInt8`? too small?

Reply:
Maximum 100 different digests, each digest corresponds to 127 histories, each history has a `sampleSQL` which may occupies 4K.
So the memory of summaries may occupy 100*127*4K=50M.
What's the size do you recommend?

Reply:
I have changed it to `math.MaxUint8`.